### DISCUSSION TITLE
Is there a way to generate normal resource names?

### DISCUSSION THREAD

**Discussion by XerXi** [GoogleCloudPlatform/terraformer#1386] [Q&A]
Hi,
At the moment when I run "terraformer import" it generates the resource names by prefixing and suffixing them. For example, I have a security group named "test", but after import it becomes:
resource "aws_security_group" "tfer--test_sg-00cc329b241230d51"
Instead of just:
resource "aws_security_group" "test"

Also, I noticed that dots in the resource names are translated to Unicode:
"dev.server.sg" becomes "dev-002E-server-002E-sg"

**Comment by drhetianhao** [comment]
any updates on this topic?

**Reply by slastique** [reply]
Hi 
you may do the following: https://github.com/GoogleCloudPlatform/terraformer?tab=readme-ov-file#planning:

1. Create a plan file
2. remove these "tfer--"
3. run `terraformer import`

**Comment by DanSega1** [comment]
I'm also need a solution for resource name
